_topic_maps/_topic_map.yml

```diff
@@ -2310,10 +2310,51 @@ Distros: openshift-origin,openshift-enterprise
 Topics:
 - Name: Overview of machine management
   File: index
+- Name: Machine phases and lifecycle
+  File: machine-phases-lifecycle
 - Name: Managing compute machines with the Machine API
-  Dir: creating_machinesets
+  Dir: compute_machine_management
   Distros: openshift-origin,openshift-enterprise
   Topics:
+  # - Name: About compute machine sets
+  #   File: mapi-compute-about
+  # - Name: Getting started with compute machine sets
+  #   File: mapi-compute-getting-started
+  - Name: Managing compute machines with compute machine sets
+    Dir: mapi_compute_managing_machines
+    Topics: 
+    - Name: Manually scaling a compute machine set
+      File: manually-scaling-machineset
+    - Name: Modifying a compute machine set
+      File: modifying-machineset
+  # - Name: Compute machine set configuration
+  #   File: mapi-compute-configuration
+  # - Name: Configuration options for compute machines
+  #   Dir: mapi_compute_provider_configurations
+  #   Topics:
+  #   - Name: Compute configuration options for Amazon Web Services
+  #     File: mapi-compute-config-options-aws
+  #   - Name: Compute configuration options for Microsoft Azure
+  #     File: mapi-compute-config-options-azure
+  #   - Name: Compute configuration options for Google Cloud Platform
+  #     File: mapi-compute-config-options-gcp
+  #   - Name: Compute configuration options for Nutanix
+  #     File: mapi-compute-config-options-nutanix
+  #   - Name: Compute configuration options for Red Hat OpenStack Platform
+  #     File: mapi-compute-config-options-openstack
+  #   - Name: Compute configuration options for VMware vSphere
+  #     File: mapi-compute-config-options-vsphere
+  - Name: Applying autoscaling to a cluster
+    File: applying-autoscaling
+  # - Name: Compute resiliency and recovery
+  #   File: mapi-compute-resiliency
+  # - Name: Troubleshooting the compute machine set
+  #   File: mapi-compute-troubleshooting
+  # - Name: Disabling compute machine sets
+  #   File: mapi-compute-disabling
```

Comment:
Commented out assemblies and directories will be uncommented and added in a later stage